examples/heat1d/CMakeLists.txt

```diff
@@ -15,27 +15,41 @@ set(heat_mpi_examples
     heat1d_pfasst
 )
 
-if(${pfasst_WITH_MPI})
-    set(all_heat_examples ${heat_examples} ${heat_mpi_examples})
+if(NOT ${pfasst_WITH_MPI})
+    foreach(example ${heat_examples})
```

Comment:
that variable `all_heat_examples` is used to specify the example binaries to be installed into the target directory. it's empty at the end of this file.